pkg/compactor/blocks_cleaner.go

```diff
@@ -159,6 +162,10 @@ func NewBlocksCleaner(
 			Name: "cortex_bucket_parquet_blocks_count",
 			Help: "Total number of parquet blocks in the bucket. Blocks marked for deletion are included.",
 		}, commonLabels),
+		tenantParquetUnConvertedBlocks: promauto.With(reg).NewGaugeVec(prometheus.GaugeOpts{
+			Name: "cortex_bucket_parquet_unconverted_blocks_count",
+			Help: "Total number of parquet blocks in the bucket. Blocks marked for deletion are included.",
```

Comment:
```suggestion
			Help: "Total number of unconverted parquet blocks in the bucket. Blocks marked for deletion are included.",
```
How about adding `unconverted`?